frontend/src/components/periodical-evaluation/references/PeriodicalEditionReferenceInput.vue

```diff
@@ -297,261 +298,331 @@ onMounted(async () => {
-          aria-label="Klammernzusatz"
-          :has-error="slotProps.hasError"
-          size="medium"
-          @blur="validateRequiredInput(reference)"
-          @focus="validationStore.remove('referenceSupplement')"
-        ></TextInput>
-      </InputField>
-    </div>
-
-    <div v-if="!isSaved" id="documentationUnit" class="flex flex-col gap-24">
       <div class="flex justify-between gap-24">
-        <InputField
-          id="courtInput"
-          v-slot="slotProps"
-          label="Gericht"
-          :validation-error="validationStore.getByField('court')"
-        >
-          <ComboboxInput
-            id="courtInput"
-            v-model="relatedDocumentationUnit.court"
-            aria-label="Gericht"
-            clear-on-choosing-item
-            :has-error="slotProps.hasError"
-            :item-service="ComboboxItemService.getCourts"
-            :read-only="reference?.documentationUnit?.hasForeignSource"
-            @focus="validationStore.remove('court')"
+        <div id="citationInputField" class="flex w-full flex-col">
+          <InputField
+            v-if="!isSaved"
+            id="citation"
+            v-slot="slotProps"
+            label="Zitatstelle *"
+            :validation-error="validationStore.getByField('citation')"
           >
-          </ComboboxInput>
-        </InputField>
-        <InputField
-          id="decisionDate"
-          v-slot="slotProps"
-          label="Entscheidungsdatum"
-          :validation-error="validationStore.getByField('decisionDate')"
-          @update:validation-error="
-            (validationError: any) =>
-              updateDateFormatValidation(validationError)
-          "
-        >
-          <DateInput
-            id="decisionDate"
-            v-model="relatedDocumentationUnit.decisionDate"
-            aria-label="Entscheidungsdatum"
-            class="ds-input-medium"
-            :has-error="slotProps.hasError"
-            :read-only="reference?.documentationUnit?.hasForeignSource"
-            @focus="validationStore.remove('decisionDate')"
-            @update:validation-error="slotProps.updateValidationError"
-          ></DateInput>
-        </InputField>
-      </div>
-      <div class="flex justify-between gap-24">
+            <div class="flex flex-grow flex-row gap-16">
+              <TextInput
+                id="citation prefix"
+                v-model="prefix"
+                aria-label="Zitatstelle Präfix"
+                placeholder="Präfix"
+                read-only
+                size="medium"
+              ></TextInput>
+              <TextInput
+                id="citation"
+                v-model="reference.citation"
+                aria-label="Zitatstelle *"
+                :has-error="slotProps.hasError"
+                placeholder="Variable"
+                size="medium"
+                @blur="validateRequiredInput(reference)"
+                @focus="validationStore.remove('citation')"
```

Comment:
@leonie-koch When testing, I noticed that the required input for literature references (including citation) are not validated correctly. However, in the code it looks all good